doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -56,6 +58,11 @@ server {
     {
         add_header 'Access-Control-Allow-Origin' '*';
         alias /usr/share/jitsi-meet/$1/$2;
+
+        # cache all versioned files
+        if ($arg_v) {
+          expires 1y;
+        }
```

Comment:
this I like, just tested it and it works great.

Reply:
Nice, TIL!